sonaranalyzer-dotnet/its/expected/Nancy/Nancy.Hosting.Self.Tests-{CA24ED85-DD68-4C10-B80A-D81C6745FCB8}-S4005.json

```diff
@@ -0,0 +1,17 @@
+{
+"issues":  [
+{
+"id":  "S4005",
+"message":  "Call the overload that takes a 'System.Uri' as an argument instead.",
+"location":  {
+"uri":  "Nancy\src\Nancy.Hosting.Self.Tests\NancySelfHostFixture.cs",
```

Comment:
FYI, this is the flagged code:
                var reader =
                    new StreamReader(WebRequest.Create(BaseUri.ToString().TrimEnd('/')).GetResponse().GetResponseStream());